packages/core-typings/src/Apps.ts

```diff
@@ -49,6 +49,7 @@ export type App = {
 	iconFileContent: string;
 	installed?: boolean;
 	isEnterpriseOnly?: boolean;
+	isSubscribed: string;
```

Comment:
This should be a boolean...

Reply:
Oh lord, that was totally an oversight. Pushing a fix commit right now.

Reply:
Solved.